recipes/cmake/3.x.x/conanfile.py

```diff
@@ -84,21 +89,33 @@ def _configure_cmake(self):
         return self._cmake
 
     def build(self):
-        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
-                              "project(CMake)",
-                              "project(CMake)\ninclude(\"{}/conanbuildinfo.cmake\")\nconan_basic_setup(NO_OUTPUT_DIRS)".format(
-                                  self.install_folder.replace("\\", "/")))
-        if self.settings.os == "Linux":
-            tools.replace_in_file(os.path.join(self._source_subfolder, "Utilities", "cmcurl", "CMakeLists.txt"),
-                                  "list(APPEND CURL_LIBS ${OPENSSL_LIBRARIES})",
-                                  "list(APPEND CURL_LIBS ${OPENSSL_LIBRARIES} ${CMAKE_DL_LIBS} pthread)")
-        cmake = self._configure_cmake()
-        cmake.build()
+        if self.options.bootstrap:
+            with tools.chdir(self._source_subfolder):
+                self.run(['./bootstrap', '--prefix={}'.format(self.package_folder), '--parallel={}'.format(tools.cpu_count())])
```

Comment:
I would have expect that to be part of the helper 🤔 https://github.com/conan-io/conan/search?p=1&q=parallel&type=code it's there for CMake and MSBuild

Reply:
Could you elaborate? I don't get it.

Reply:
Derp, sorry that was really unclear sorry (it made sense it my head I swear 😆 )

The option for `--parallel` is handled by other build helpers I was surprised to see it here.

Reply:
Ok, thanks.
I'm not sure on how to reuse the helpers here.
The bootstrapping is not using the normal CMake flow but this custom shell script as entrypoint.